### PR TITLE
Add a page with links to the pycharm setup github pages

### PR DESCRIPTION
**Pull request by cassm**
As pycharm benefits from quite a bit of specific setup, I think it's a good idea to link to the github wiki for detailed instructions. We could potentially create a pycharm settings file that people could just import, but that seems like it could go far wrong if JB changed something.

### CODE REVIEW

**Review by hawaiikaos — APPROVED** [review]
LGTM

File: practices/text-editors/setting-up-pycharm.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: Setting Up Pycharm
+---
+
+* [General Pycharm setup](https://github.com/wegotpop/popsss/wiki/Setting-Up-Pycharm-including-black-and-isort)
```
Comment (Amwam) [inline practices/text-editors/setting-up-pycharm.md:6]: We can't link directly to the wiki here as the principles and practices website is a public site

Comment (cassm) [reply practices/text-editors/setting-up-pycharm.md:6]: That's a good point. I could create a "Setting up pycharm" parent page on the wiki and put instructions for finding it on the wiki? Something like `This process is covered in the "Setting Up Pycharm" page in the popsss wiki`.

Comment (tcowpop) [reply practices/text-editors/setting-up-pycharm.md:6]: Would it make more sense to move all IDE setup guides under our wiki instead so they are easier to keep up to date?

**Comment by cassm** [comment]
> I am not sure **we prefer** is the right wording as I think the company does not care what editor you use 😄 .

I see what you mean, but there are issues with the community version not supporting remote interpreters, so it adds fragility to dependency management. My understanding from the SIG was that people should be using the professional version, although I may have got the wrong end of the stick.

**Comment by StaniaWeGotPop** [comment]
> > I am not sure **we prefer** is the right wording as I think the company does not care what editor you use 😄 .
> 
> I see what you mean, but there are issues with the community version not supporting remote interpreters, so it adds fragility to dependency management. My understanding from the SIG was that people should be using the professional version, although I may have got the wrong end of the stick.

I might be wrong then. 
as we cannot link to our wiki in this repo, is it even worth creating this page here? instead we could ensure the links are clearly visible in the wiki.

**Comment by cassm** [comment]
> > > I am not sure **we prefer** is the right wording as I think the company does not care what editor you use 😄 .
> > 
> > 
> > I see what you mean, but there are issues with the community version not supporting remote interpreters, so it adds fragility to dependency management. My understanding from the SIG was that people should be using the professional version, although I may have got the wrong end of the stick.
> 
> I might be wrong then. as we cannot link to our wiki in this repo, is it even worth creating this page here? instead we could ensure the links are clearly visible in the wiki.

That's a good point. The issue this was intended to fix was that it was difficult to find pycharm setup information during onboarding, so maybe it just needs adding to the trello/whatever we replace the trello with

File: practices/text-editors/setting-up-pycharm.md (hunk 1 above)
Comment (cassm) [reply practices/text-editors/setting-up-pycharm.md:6]: That sounds like a good plan. I can do that today.

**Comment by cassm** [comment]
I'll make a page on the wiki collecting the IDE setup information that's currently here, and we can link to that from the onboarding board instead

**Pull request closed without merging**